leveldb-library.podspec

```diff
@@ -0,0 +1,61 @@
+Pod::Spec.new do |s|
+  s.name         =  'leveldb-library'
+  s.version      =  '1.22.3'
+  s.license      =  'New BSD'
+  s.summary      =  'A fast key-value storage library '
+  s.description  =  'LevelDB is a fast key-value storage library written at Google that provides ' +
+                    'an ordered mapping from string keys to string values.'
+  s.homepage     =  'https://github.com/google/leveldb'
+  s.authors      =  'The LevelDB Authors'
+
+  s.ios.deployment_target = '11.0'
+  s.osx.deployment_target = '10.13'
+  s.tvos.deployment_target = '11.0'
+  s.watchos.deployment_target = '6.0'
+
+  s.source       =  {
+    :git => 'https://github.com/firebase/leveldb.git',
+    :tag => s.version.to_s
```

Comment:
@paulb777, can the pod share the same tagging convention that SPM uses? My understanding is that I need to tag the pod and push it staging, but I don't want to tag with a convention that will _release_ it on spm.

Reply:
I'd prefer not to share the tagging and instead use the CocoaPods prefix that Firebase podspecs do. This will enable us to stage the podspec on SpecsStaging without publishing an SPM release.